### PR TITLE
[UI] Login page

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 

## Description
This PR is based [on another PR](https://github.com/mautic/mautic/pull/13735), where I introduce tokens to replace most of the UI variables, aiming to improve consistency in all components. **We need it to be merged before this one becomes available for code review.**

This PR creates a new login page.

| Before                                 | After
| -------------------------------------- | ---
| image                                        | image

Basic prototype, considering changes to fields and buttons:

Desktop - 1

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. You don't even need to login, just see

### CODE REVIEW

File: app/bundles/UserBundle/Resources/views/Security/base.html.twig [hunk 1]
```diff
@@ -14,33 +14,98 @@
...
                     </div>
                 </div>
+                <div class="row gap-sm fw-nowrap">
+                    <div class="col-lg-3 jc-space-between gap-lg background-brand-alt pa-32 features-container">
+                        <img class="logo" src="/app/assets/images/horizontal_logo.svg"></img>
+                        <div class="fs-heading-03">{{ 'mautic.user.auth.card.features'|trans }}</div>
+                    </div>
+            
+                    <div class="col-lg-3 jc-end background-brand pa-32 features-container fg-1">
+                        <svg class="circles" width="210" height="120" viewBox="0 0 210 120" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (shinde-rahul) [inline app/bundles/UserBundle/Resources/views/Security/base.html.twig:35]: A suggestion: we should move all the SVG to their own twig file and include them wherever intended.

File: app/bundles/UserBundle/Resources/views/Security/base.html.twig [hunk 2]
```diff
@@ -14,33 +14,98 @@
...
+                        <ul class="ds-list-group ds-list-none">
+                            <li>{{ 'mautic.user.auth.card.features.email_marketing'|trans }}<i class="ri-mail-send-line"></i></li>
+                            <li>{{ 'mautic.user.auth.card.features.social_monitoring'|trans }}<i class="ri-chat-1-line"></i></li>
+                            <li>{{ 'mautic.user.auth.card.features.forms'|trans }}<i class="ri-survey-line"></i></li>
+                            <li>{{ 'mautic.user.auth.card.features.landing_pages'|trans }}<i class="ri-pages-line"></i></li>
+                        </ul>
+                    </div>
+                    
+                    <div class="col-lg-6 jc-end background-brand pa-32 features-container">
+                        <svg class="target" width="509" height="381" viewBox="0 0 509 381" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (shinde-rahul) [inline app/bundles/UserBundle/Resources/views/Security/base.html.twig:82]: Same as suggested above.

File: app/bundles/UserBundle/Resources/views/Security/login.html.twig [hunk 3]
```diff
@@ -15,38 +15,52 @@
...
+				<div class="mt-sm text-right">
+					<a href="{{ path('mautic_user_passwordreset') }}">{% trans %}mautic.user.user.passwordreset.link{% endtrans %}</a>
+				</div>
+			</form>
 		</div>
-	</form>
+	</div>
+
+
+
```
Comment (shinde-rahul) [inline app/bundles/UserBundle/Resources/views/Security/login.html.twig:61]:
```suggestion

```

**Comment by andersonjeccel** [comment]
@shinde-rahul What did you think about the proposal, in general?

This PR is a first suggestion of bringing Mautic closer to other apps in the market, I'd like ideas to improve

File: app/bundles/UserBundle/Resources/views/Security/base.html.twig (hunk 1 above)
Comment (andersonjeccel) [reply app/bundles/UserBundle/Resources/views/Security/base.html.twig:35]: One twig file for each SVG? Or one for both?

**Comment by shinde-rahul** [comment]
@andersonjeccel, The idea is to have the elements in their designated place to help identify them better and make the reusable supper cool. I was thinking more about the developer experience.

File: app/bundles/UserBundle/Resources/views/Security/base.html.twig (hunk 1 above)
Comment (shinde-rahul) [reply app/bundles/UserBundle/Resources/views/Security/base.html.twig:35]: Separate file for each SVG. The file name can add readability.

**Comment by rabiibrahimi** [comment]
Why are such good PR which enhance the UI and UX is closed instead of being merged ??

**Pull request closed without merging**